### PR TITLE
qemu_disk_img: Fix no matching bus on aarch64

### PR DESCRIPTION
**Pull request by zhenyzha**
Fix no matching bus on aarch64

ID: 1931031
Signed-off-by: Zhenyu Zhang <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
(1/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.aarch64.io-github-autotest-qemu.qemu_disk_img.info.change_block_interface.arm64-pci: PASS (384.97 s)
 (2/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.aarch64.io-github-autotest-qemu.qemu_disk_img.info.ch
ange_block_interface.arm64-pci: PASS (438.40 s)
@PaulYuuu Could you pls help review, thanks.

File: qemu/tests/cfg/qemu_disk_img.cfg [hunk 1]
```diff
@@ -153,6 +153,11 @@
                             drive_format_snA = virtio
                         virtio_blk:
                             drive_format_snA = scsi-hd
+                    aarch64:
```
Comment (PaulYuuu) [inline qemu/tests/cfg/qemu_disk_img.cfg:156]: It can be appended at L151
```
pseries, aarch64:
```

Comment (zhenyzha) [reply qemu/tests/cfg/qemu_disk_img.cfg]: Thanks for the reminder.

**Comment by zhenyzha** [comment]
@PaulYuuu Hi could you pls help review again? thanks.

**Comment by PaulYuuu** [comment]
Missing bug ID.

**Comment by zhenyzha** [comment]
> Missing bug ID.

Corrected, sorry for the inconvenience.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**